packages/salesforcedx-vscode-core/src/constants.ts

```diff
@@ -33,3 +33,4 @@ export const TELEMETRY_ENABLED = 'telemetry.enabled';
 export const USE_APEX_LIBRARY = 'experimental.useApexLibrary';
 export const FUNCTIONS_ENABLED = 'experimental.supportFunctions';
 export const USE_TEMPLATES_LIBRARY = 'experimental.useTemplatesLibrary';
+export const FUNCTIONS_PULL_DEPENDENCIES = 'functions-pull-dependencies';
```

Comment:
Let's have this be `experimental.functions...` since the other functions flag is also has the `experimental` prefix